<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cresemo</class>
 <widget class="QMainWindow" name="Cresemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cresemo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/application.png</normaloff>:/icons/application.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Select CVS Repository</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refreshBN">
          <property name="toolTip">
           <string>Refresh Directories (F5)</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="shortcut">
           <string>F5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="fileBrowser">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="folderLabel">
          <property name="text">
           <string>Root Folder :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="currentFolder">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="cvsPropertiesGB">
        <property name="font">
         <font>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>CVS Properties</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Username :</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="usernameLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Server : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="serverLE">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Apply Recursively :</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="recursiveCB">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="applyBN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Apply Changes</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/icons/accept.png</normaloff>:/icons/icons/accept.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Messages :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="infoWindow">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>fileBrowser</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Cresemo</receiver>
   <slot>folderSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usernameLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Cresemo</receiver>
   <slot>usernameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverLE</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Cresemo</receiver>
   <slot>serverChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>applyBN</sender>
   <signal>clicked()</signal>
   <receiver>Cresemo</receiver>
   <slot>applyChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshBN</sender>
   <signal>clicked()</signal>
   <receiver>Cresemo</receiver>
   <slot>refreshDirectories()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>folderSelected(QModelIndex)</slot>
  <slot>usernameChanged(QString)</slot>
  <slot>serverChanged(QString)</slot>
  <slot>applyChanges()</slot>
  <slot>refreshDirectories()</slot>
 </slots>
</ui>
